crates/sui-core/src/authority.rs

```diff
@@ -344,6 +344,10 @@ impl AuthorityState {
         self.committee.load().epoch
     }
 
+    pub fn clone_epoch_store(&self) -> Arc<EpochStore> {
```

Comment:
Returning a `&Arc` is my usual way of doing this to let the called decide what they need.